peer.go

```diff
@@ -449,11 +449,10 @@ func (p *peer) Start() error {
 // initGossipSync initializes either a gossip syncer or an initial routing
 // dump, depending on the negotiated synchronization method.
 func (p *peer) initGossipSync() {
-	switch {
 
 	// If the remote peer knows of the new gossip queries feature, then
 	// we'll create a new gossipSyncer in the AuthenticatedGossiper for it.
-	case p.remoteFeatures.HasFeature(lnwire.GossipQueriesOptional):
+	if p.remoteFeatures.HasFeature(lnwire.GossipQueriesOptional) {
```

Comment:
M-m-muh switch statements! 

Solid change though, can be upgraded back once we actually have additional cases.